qa/L0_s3_local/test.sh

```diff
@@ -222,6 +222,75 @@ wait $SERVER_PID
 awslocal $ENDPOINT_FLAG s3 rm s3://demo-bucket1.0 --recursive --include "*" && \
     awslocal $ENDPOINT_FLAG s3 rb s3://demo-bucket1.0
 
+# Test for multiple model stores using S3 cloud repositories
+BACKENDS1="graphdef libtorch"
+BACKENDS2="onnx plan savedmodel"
+BACKENDS="$BACKENDS1 $BACKENDS2"
+
+rm -rf models1 && mkdir models1
+for BACKEND in $BACKENDS1; do
+    cp -r $DATADIR/${BACKEND}_float32_float32_float32 models1/.
+    # Remove version policy from config.pbtxt
+    sed -i '/^version_policy/d' models1/${BACKEND}_float32_float32_float32/config.pbtxt
+done
+
+rm -rf models2 && mkdir models2
+for BACKEND in $BACKENDS2; do
+    cp -r $DATADIR/${BACKEND}_float32_float32_float32 models2/.
+    # Remove version policy from config.pbtxt
+    sed -i '/^version_policy/d' models2/${BACKEND}_float32_float32_float32/config.pbtxt
+done
+
+BUCKET_NAME="demo-bucket"
+for BUCKET_SUFFIX in 1 2; do
+    # Cleanup bucket if exists
+    awslocal $ENDPOINT_FLAG s3 rm s3://$BUCKET_NAME$BUCKET_SUFFIX --recursive --include "*" && \
+        awslocal $ENDPOINT_FLAG s3 rb s3://$BUCKET_NAME$BUCKET_SUFFIX || true
+
+    # Create and add data to bucket
+    awslocal $ENDPOINT_FLAG s3 mb s3://$BUCKET_NAME$BUCKET_SUFFIX && \
+        awslocal $ENDPOINT_FLAG s3 sync models$BUCKET_SUFFIX s3://$BUCKET_NAME$BUCKET_SUFFIX
+done
+
+SERVER_ARGS="--model-repository=s3://localhost:4572/demo-bucket1 --model-repository=s3://localhost:4572/demo-bucket2 --model-control-mode=explicit --log-verbose=1"
```

Comment:
Can we generate "--model-repository=s3://localhost:4572/demo-bucket1 --model-repository=s3://localhost:4572/demo-bucket2" part of the SERVER_ARGS from the loop above?